tensorflow_datasets/image/lfw.py

```diff
@@ -40,81 +40,83 @@
 
 
 class LFW(tfds.core.GeneratorBasedBuilder):
-    URL = "http://vis-www.cs.umass.edu/lfw/#resources"
-    VERSION = tfds.core.Version("2.0.0")   
-    def _info(self):
-        return tfds.core.DatasetInfo(
-            builder=self,
-            description=("Labeled Faces in the Wild:\
-             A Database for Studying Face Recognition in Unconstrained Environments"),
-            features=tfds.features.FeaturesDict({
-              "anchor": tfds.features.Image(shape=LFW_IMAGE_SHAPE),
-              "positive": tfds.features.Image(shape=LFW_IMAGE_SHAPE),
-              "negative": tfds.features.Image(shape=LFW_IMAGE_SHAPE),
+  """LFW Class"""
+  URL = "http://vis-www.cs.umass.edu/lfw/#resources"
+  VERSION = tfds.core.Version("2.0.0")
```

Comment:
Why do you start from version 2.0.0 ?
Learn more about versioning by checking other datasets commits.

Reply:
Changed the version to 0.1.0